src/EnergyPlus/HVACManager.cc

```diff
@@ -2202,26 +2104,24 @@ void ResolveLockoutFlags(EnergyPlusData &state, bool &SimAir) // TRUE means air
     // METHODOLOGY EMPLOYED:
     // Checks if loop lockout flags are .TRUE.; if so, sets SimAirLoops to .TRUE.
 
-    auto &AirLoopControlInfo = state.dataAirLoop->AirLoopControlInfo;
-
```

Comment:
[src/EnergyPlus/HVACManager.cc:2205]:(style),[constVariable],Variable 'AirLoopControlInfo' can be declared as reference to const